pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -566,6 +575,9 @@ func (o *RoleModificationOptions) AddRole() error {
 	if err != nil {
 		return err
 	}
+	if isUpdate {
+		o.checkRolebindingAutoupdate(roleBinding)
```

Comment:
Reconciliation on master start never removes anything so not sure if we want to warn here.

Reply:
Uhmm I guess we could drop this, then I'll change the test to check on the delete op ...
I wasn't sure if there were any conceivable scenario where a reconcile after add could still cause issues ...

Reply:
While I would love to say "you cannot mess with the default RBAC stuff at all because we will force it back exactly on server start," doing so is basically guaranteed to break people (especially since the old commands re-used existing objects).  So for the time being we are stuck with "just add back anything that was removed."

> I wasn't sure if there were any conceivable scenario where a reconcile after add could still cause issues ...

That would be considered a bug in the reconcile code.